src/main/java/com/auth0/client/mgmt/JobsEntity.java

```diff
@@ -21,11 +21,32 @@
         super(client, baseUrl, apiToken);
     }
 
+    /**
+     * Request a Job. A token with scope create:users is needed.
+     * See https://auth0.com/docs/api/management/v2#!/Jobs/get_jobs_by_id
+     *
+     * @param jobId the id of the job to retrieve.
+     * @return a Request to execute.
+     */
+    public Request<Job> get(String jobId) {
+        Asserts.assertNotNull(jobId, "job id");
+
+        HttpUrl.Builder builder = baseUrl
+                .newBuilder()
+                .addPathSegments("api/v2/jobs")
+                .addPathSegment(jobId);
+        String url = builder.build().toString();
```

Comment:
Existing one function in this class create the url in a single step without holding on to a builder property - worth doing that here as well given it's not re-used?